bgpd/bgp_pbr.c

```diff
@@ -2684,6 +2684,7 @@ static void bgp_pbr_handle_entry(struct bgp *bgp, struct bgp_path_info *path,
 	struct bgp_pbr_or_filter bpof;
 	struct bgp_pbr_val_mask bpvm;
 
+	memset(&range, 0, sizeof(range));
```

Comment:
What about range_icmp_code as well?

Reply:
clang sa is not complaining about it.  so I'm not worried